server/dining.py

```diff
@@ -89,20 +89,27 @@ def get_data():
                         get_data)
 
 
-@app.route('/dining/preferences', methods=['POST'])
-def save_dining_preferences():
+@app.route('/dining/preferences/V2', methods=['POST'])
```

Comment:
We should ensure that the android app doesn't rely on this endpoint before changing it.

Reply:
cool I'm pretty sure they don't